lib/customer.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'restaurant.rb'
+
+class Customer
+
+  attr_accessor :order
+
+  def intialize
+    order =  []
+  end
+
+  def checks_menu(restaurant)
+    restaurant.menu.each do |dish, price|
+      puts "#{dish} costs #{price}"
```

Comment:
By using puts your tying the presentation of your takeaway system to terminal. What if I wanted this string to be printed onto a web page?